releases/v0.8.0-rc1.yaml

```diff
@@ -0,0 +1,28 @@
+---
+version: v0.8.0-rc1
+name: 0.8.0-rc1
+release-notes: |
```

Comment:
These release notes are for the product release - this is the shipyard release.  I think we just should omit RNs here and add them manually later if we want.  In the past I/we generally have not added RNs for the projects.